### PR TITLE
Update inspector docs

### PR DESCRIPTION
**Pull request by alex-tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…/start-using-inspector**](https://docs-qofytcwwb.avo-dev.app/data-design/start-using-inspector) | [**/workspace/inspector**](https://docs-qofytcwwb.avo-dev.app/workspace/inspector) |
|:-:|:-:|
|<a href="https://docs-qofytcwwb.avo-dev.app/data-design/start-using-inspector">Screenshot of /data-design/start-using-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/UW43Xnrc4c1qzxeTZhqpvqx84YWQgZDusu1l_ofJlz70hxJbFpw0W0nitQDtQMsId921-cvLKc_9SmFN3stHhA.png">(view full size)</a> |<a href="https://docs-qofytcwwb.avo-dev.app/workspace/inspector">Screenshot of /workspace/inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/R_jcMNP1V7NQaPfqbUiw9yCOyqOoe8eAO7wIMm1_yWcwWq-NPdRSCHaQJty7UJ6xJEkked2iqxPyyb54ysAimg.png">(view full size)</a> |







Commit f05db87fddfa27f09a86c8889c16e9e07c8c658b (https://docs-qofytcwwb.avo-dev.app).

File: pages/data-design/start-using-inspector.mdx [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 ---
 
 import Link from '../../components/Link';
 
-After setting up your sources to send data to the Avo Inspector with the <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector SDKs</a></Link> you will see the data appearing in the <Link passHref href="/workspace/inspector"><a>the Inspector tab</a></Link> tab.
+After setting up your sources to send data to the Avo Inspector with the <Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Avo Inspector SDKs</a></Link> you will see the data appearing in the the Inspect tab.
 
-Open the <Link passHref href="/workspace/inspector#default-view-and-switching-between-views">Inspector events screen view</Link>, browse the list or search for specific event, select it and the right panel will appear with detailed information about the health of the given event.
+<Link passHref href="/workspace/inspector">Inspect tab description</Link>
+<p/>
```
Comment (hugihlynsson) [inline pages/data-design/start-using-inspector.mdx:11]: Should this closing p be here?

File: pages/data-design/start-using-inspector.mdx [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 
-From there you would be able to analyze the problem, change event description in the tracking plan or add it to the tracking plan if it is not there yet.
+Open the <Link passHref href="/workspace/inspector#Overview">Inspector overview</Link>, browse the list or search for specific event, select it and the right panel will appear with detailed information about the health of the given event.
+
+From there you would be able to analyze the problems with the given event, if any, change event description in the tracking plan or add it to the tracking plan if it is not there yet.
+
+<Link href="/explore-tracking-plan/issue-types-in-inspector" passHref><a>List of issue types Inspector can detect</a></Link>
+<p/>
```
Comment (hugihlynsson) [inline pages/data-design/start-using-inspector.mdx:18]: Also this /p?

**Review by hugihlynsson — CHANGES_REQUESTED** [review]
I noticed two stray closing p tags but otherwise the changes look great 👌

File: pages/data-design/start-using-inspector.mdx (hunk 1 above)
Comment (alex-tpom6oh) [reply pages/data-design/start-using-inspector.mdx:11]: Yeah, I'm not sure why double space does not create a new line 🤔 But the way to solve this should be <br/>, not <p/>

**Pull request merged**